lib/acts_as_xapian/acts_as_xapian.rb

```diff
@@ -936,9 +936,17 @@ def xapian_index(terms = true, values = true, texts = true)
         if value.kind_of?(Array)
           for v in value
             doc.add_term(term[1] + v)
+
+            if term[1] == 'N'
+              doc.add_posting(term[1] + v, 1, 50)
+            end
```

Comment:
Feel a bit uneasy about hard coding the `N` and `50` here. Its probably okay as this library is included in the repo. I took a look at how we could do this differently and came up with: https://gist.github.com/gbp/800309a5349ebbdcfba7ef1d58cb795b which seems to work. Thoughts?